tests/test_security_stop_report.py

```
@@ -0,0 +1,71 @@
+        stop_reason=malicious_reason,
+        stopped_at="2023-10-27T10:00:00Z"
+    )
+
+    state = {
+        "flow_id": "test-flow",
+        "status": "stopping",
+        "completed_steps": [],
+        "pending_steps": []
+    }
+
+    async def run_test():
```

Ruff (0.15.0)

[warning] Missing return type annotation for private function `run_test`

Add return type annotation: `None`

(ANN202)